src/applications/check-in/day-of/pages/NextOfKin.jsx

```diff
@@ -9,8 +9,8 @@ import BackButton from '../components/BackButton';
 import BackToHome from '../components/BackToHome';
 import { focusElement } from 'platform/utilities/ui';
 import Footer from '../components/Footer';
-import DemographicItem from '../../components/DemographicItem';
 import { seeStaffMessageUpdated } from '../actions';
+import NextOfKinDisplay from '../../components/pages/nextOfKin/NextOfKinDisplay';
```

Comment:
Since these pages were touched, we should use the `<va-loading-indicator/>` tag instead of the component.

Reply:
all set. thanks for the help with that.

Reply:
also updated in Demographics day of